pages/Hypr Ecosystem/hyprlock.md

```diff
@@ -47,7 +47,16 @@ widget_name {
 }
 ```
 
-`monitor` can be left empty for "all monitors"
+The `monitor` could be either the __port name__ or __description__ value from
+the output of `hyprctl monitors` command. It can be left empty for "all
+monitors"
+
+{{< callout >}}
+
+Pay attention! You do not need to add `desc:` prefix like in
+[`hyprland.conf`](../Configuring/Monitors.md)
+
+{{</ callout >}}
```

Comment:
That is not true.

The current check is `surf->output->stringDesc.starts_with(c.monitor)`

Reply:
Should we support prefixing with `desc:` as well for consistency?

Reply:
prolly

Reply:
The `desc:` variant stops work for me at all. The ripgrep did not find any occurrences of `desc:` in sources. I found the line @PaideiaDilemma talking about in sources and tries to [remove](https://github.com/ein-shved/nixpkgs/commit/9dc25e19df43443ea5e29155709cf379aa50f4ee#diff-61d0726a847c32c85fc9da9e8ea05c47a50c6740d263d05b889697981527fb66) the `desc:` from config. And it worked.

So I decided to update doc and mention this peculiarity.

I think, we should to add prefix for consistency too, but we should have clear actual documentation as well

Reply:
I create a MR in hyprlock to support `desc:`.

Sorry I initially misunderstood the wording, because i read it too fast.

Reply:
I think you can remove the this section, but the rest is fine.